app/frontend/src/components/Bar/BarWithHelp.tsx

```diff
@@ -7,28 +8,39 @@ import ScoreBar from "./ScoreBar";
 
 const useStyles = makeStyles((theme) => ({
   button: {
-    padding: theme.spacing(1),
+    padding: 0,
+    paddingLeft: theme.spacing(1),
   },
+
   root: {
     alignItems: "center",
     display: "flex",
   },
+
+  marginBottom2: {
+    marginBottom: theme.spacing(2),
+  },
 }));
 
 interface BarWithHelpProps {
   value: number;
   label: string;
   description: string;
+  mb2?: boolean;
 }
```

Comment:
Use `className` - we aren't using utility classes in general and `className` is more flexible

Reply:
unitless line-height is pretty messy to use https://stackoverflow.com/questions/12303169/line-height-without-units#answer-12304722 and an lead to lots of layouts problems difficult to debug. When set to a fixed unit it makes things easier. In this context, the line-height is fixed by design, and it will change if design changes, imo it's way better to set it with units.